app/models/agents/java_script_agent.rb

```diff
@@ -114,12 +114,9 @@ def execute_js(js_function, incoming_events = [])
       context["getOptions"] = lambda { |a, x| interpolated.to_json }
       context["doLog"] = lambda { |a, x| log x }
       context["doError"] = lambda { |a, x| error x }
-      context["getMemory"] = lambda do |a, x, y|
```

Comment:
As handling of variable length arguments is awkward (*) in V8 (and JavaScript itself), I'd rather have a separate functions for getting the whole memory and an individual key value.

* Ruby's `lambda` has a stricter argument length check compared to `proc`, but V8::Context seems to adjust arguments on the caller side, which gives me a confused feeling.